plans/sep6.md

```diff
@@ -0,0 +1,213 @@
+## SEP-6 withdraw and deposit
+
+Deposit and withdraw share a good number of steps and Transferresponse shapes,
+meaning most of the API can be consistent across both.
+
+### Types
+
+```ts
+type startWithdrawal = (transferServer: string) => PendingWithdrawal;
```

Comment:
I'm not sure if this is a common convention, but I like all types, interfaces and enums to be written in PascalCase.

Reply:
(I could be corrected on this.)

Reply:
This is more a function stub than a real type.